nmostesting/suites/IS0501Test.py

```diff
@@ -477,6 +477,18 @@ def test_23(self, test):
         else:
             return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_23_01(self, test):
+        """Senders accept a patch request with empty leg(s) in transport parameters"""
+
+        if len(self.receivers) > 0:

```

Comment:
`if len(self.senders) > 0` :)

Reply:
Obviously I was just checking you were paying attention 🙄 
Thanks!!